### ISSUE TITLE
Enhanced Visuals render on HUD

### ISSUE THREAD

**Issue by heiligerkek** [ferriarnus/VivecraftForge#331]
The visual effects from the mod "enhanced visuals" render on the hud on your left arm

**Comment by fayer3** [comment]
yeah, not much we can do about that I think

**Comment by markerz0** [comment]
I'm having same issue, can't you have a toggle for only hotbar renders on left arm and all other stuff defaults to viewport?

**Comment by fayer3** [comment]
no, since all stuff that render after the level can be either a post process effect, or actual UI elements.
there is no way to distinguish them automatically

**Comment by fayer3** [comment]
in the future there might be a way to separate them manually though. no eta on that though
